src/store/modules/inbox.js

```diff
@@ -3,17 +3,53 @@
 
 const state = {
   selectedInboxItemId: 0,
-  // other inbox state values
 };
 
 // getters
-const getters = {};
+const getters = {
+  getInboxItems(state) {
+    let inboxItems = [
+      {
+        id: 5,
+        senderIcon: "./src/assets/UserAccount/VirtualConcierge.svg",
+        senderName: "Virtual Concierge",
+        teaserText: "Hi, how can I help you?",
+        dayRead: "",
+        selected: false,
+      },
+      {
+        id: 100,
+        senderIcon: "./src/assets/UserAccount/VirtualConcierge.svg",
+        senderName: "Job Bank",
+        teaserText:
+          "You are receiving this email because you recently completed an application for Employment Insurance.",
+        dayRead: "Weds",
+        selected: false,
+      },
+    ];
+
+    let selectedItem = inboxItems.find(
+      (inboxItem) => inboxItem.id === state.selectedInboxItemId
+    );
+    if (selectedItem) selectedItem.selected = true;
+
+    return inboxItems;
+  },
+};
 
 // actions
-const actions = {};
+const actions = {
+  async selectInboxItem(context, id) {
+    context.commit("updateSelectedInboxItemId", id);
+  },
```

Comment:
Do you need to make this async? I don't think there will be an API call here. What do you think?

Reply:
It doesn't but I rather leave it for consistency sake

Reply:
I think it makes more sense to remove it, since having "async" in front of it makes it _seem_ like it's an async operation but if it isn't expected to be, then it's misleading.